addons/website_sale/static/src/js/website_sale_dashboard/website_sale_dashboard.xml

```diff
@@ -1,85 +1,98 @@
-                    t-on-click="setSearchContext"
+                    class="o_dashboard_card btn flex-grow-1 d-md-flex d-lg-block flex-md-row align-items-md-baseline justify-content-md-center gap-md-1 g-col-4 g-col-md-4 g-col-lg-3 g-col-xxl-2 py-2 px-4 lh-sm"
+                    t-att-class="this.getDashboardCardAdditionalClass('to_confirm')"
+                    t-on-click="state.eCommerceData['overall']['to_confirm'] > 0 ? setSearchContext : () => {}"
                     filter_name="to_confirm,from_website"
                 >
-                    <a href="#" class="btn p-1 p-lg-2 text-truncate text-wrap">
-                        <h2 class="m-0 fw-bold o_dashboard_card_data" t-esc="state.eCommerceData['overall']['to_confirm']"/>
-                        <h4 class="fw-semibold text-muted o_dashboard_card_text">To Confirm</h4>
-                    </a>
-                </div>
+                    <span class="o_dashboard_card_data fs-4 fw-bolder" t-esc="state.eCommerceData['overall']['to_confirm']"/>
+                    <span class="d-block">To Confirm</span>
+                </button>
 
-                <div
+                <button
                     title="Orders to Invoice"
-                    class="o_dashboard_card o_blue_card text-center rounded-3 p-2 shadow-sm flex-shrink-1"
-                    t-on-click="setSearchContext"
+                    class="o_dashboard_card btn flex-grow-1 d-md-flex d-lg-block flex-md-row align-items-md-baseline justify-content-md-center gap-md-1 g-col-4 g-col-md-4 g-col-lg-3 g-col-xxl-2 py-2 px-4 lh-sm"
+                    t-att-class="this.getDashboardCardAdditionalClass('to_invoice')"
+                    t-on-click="state.eCommerceData['overall']['to_invoice'] > 0 ? setSearchContext : () => {}"
                     filter_name="to_invoice,from_website,order_confirmed"
                 >
-                    <a href="#" class="btn p-1 p-lg-2 text-truncate text-wrap">
-                        <h2 class="m-0 fw-bold o_dashboard_card_data" t-esc="state.eCommerceData['overall']['to_invoice']"/>
-                        <h4 class="fw-semibold text-muted o_dashboard_card_text">To Invoice</h4>
-                    </a>
-                </div>
+                    <span class="o_dashboard_card_data fs-4 fw-bolder" t-esc="state.eCommerceData['overall']['to_invoice']"/>
+                    <span class="d-block">To Invoice</span>
+                </button>
             </div>
 
             <!-- Right Section -->
-            <div class="o_right_section align-content-center d-flex flex-wrap gap-4 flex-grow-1 justify-content-end">
-                <div class="o_date_filter_wrapper d-flex align-items-center w-100 w-md-auto mb-3 mb-md-0">
-                    <DateFilterButton update.bind="updateDashboardState" selectedFilter="state.selectedFilter"/>
+            <div class="d-flex flex-column flex-md-row border rounded-3 pb-2 py-md-2 w-100 w-md-auto gap-2 gap-md-0 overflow-hidden">
```

Comment:
The `overflow-hidden` here It could make us loose valuable information, do we need it?